tests/lib/rules/prefer-explicit-assert.test.ts

```diff
@@ -54,6 +54,24 @@ ruleTester.run(RULE_NAME, rule, {
     ...COMBINED_QUERIES_METHODS.map((queryMethod) => ({
       code: `const quxElement = get${queryMethod}('qux')`,
     })),
+    ...COMBINED_QUERIES_METHODS.map((queryMethod) => ({
+      code: `
+      async () => {
+        const quxElement = await find${queryMethod}('qux')
+      }`,
+    })),
+    ...COMBINED_QUERIES_METHODS.map((queryMethod) => ({
+      code: `const quxElement = find${queryMethod}('qux')`,
+    })),
+    ...COMBINED_QUERIES_METHODS.map((queryMethod) => ({
+      code: `const quxElement = screen.find${queryMethod}('qux')`,
+    })),
+    ...COMBINED_QUERIES_METHODS.map((queryMethod) => ({
+      code: `
+      async () => {
+        const quxElement = await screen.find${queryMethod}('qux')
+      }`,
+    })),
```

Comment:
Can you add an extra valid test for a function returning a promise from `findBy*` query?

```js
function findBySubmit() {
  // this is valid
  return screen.findByRole('button', { name: 'submit' })
}
```

Reply:
Oh, seems like you've found a bug 😄

Reply:
Great catch :) Should be good now